web-common/src/features/models/workspace/ModelBody.svelte

```diff
@@ -53,6 +56,14 @@
   $: runtimeInstanceId = $runtime.instanceId;
   const updateModel = createRuntimeServicePutFileAndReconcile();
 
+  let limit = 150;
```

Comment:
Let's make this `const` if we are not changing this anywhere

Reply:
Done